### PR TITLE
Prepare Docs for KKP v2.19

### LINKED ISSUE #920: Prepare docs for release v2.19
Create a v2.19 section in the docs and remove versions that are not supported anymore.

See: https://handbook.kubermatic.com/engineering/release-processes/kkp-release-handbook/#prepare-docs

### PR DESCRIPTION
**Pull request by dermorz**
Closes #920 

* [Updated compatibility matrix](https://deploy-preview-921--cranky-newton-4e6ed2.netlify.app/kubermatic/v2.19/architecture/support_policy/supported_versions/)
* [Updated components versioning table](https://deploy-preview-921--cranky-newton-4e6ed2.netlify.app/kubermatic/v2.19/architecture/support_policy/kkp_components_versioning/)
* Copied over and adjusted current content from version `master` to `v2.19`
* Restored older content for older versions to make these selectable and availabile to read

Link to the preview: https://deploy-preview-921--cranky-newton-4e6ed2.netlify.app/kubermatic/v2.19/

### CODE REVIEW

File: content/kubermatic/master/data/seed.yaml [hunk 1]
```diff
@@ -131,23 +129,6 @@ spec:
...
-          # Images to use for each supported operating system
-          images:
-            amzn2: ""
-            centos: ""
-            flatcar: ""
-            rhel: ""
-            sles: ""
-            ubuntu: ""
-          # Optional: Port to use when connecting to the Nutanix Prism Central endpoint (defaults to 9440)
-          port: 9440
```
Comment (embik) [inline content/kubermatic/master/data/seed.yaml:150]: This doesn't look right, why is it getting reverted? This was added as part of https://github.com/kubermatic/docs/commit/afaf873ed279d9b404495691a1bd0300abc0b0d7 syncing from KKP `master´.

Comment (dermorz) [reply content/kubermatic/master/data/seed.yaml:150]: This was generated by `./hack/ci/update-docs.sh` and I blindly trusted the results. I will drop this commit. Thanks!

Comment (embik) [reply content/kubermatic/master/data/seed.yaml]: Thanks for checking again!

File: content/kubermatic/v2.18/architecture/requirements/support_policy/supported_versions/_index.en.md [hunk 2]
```diff
@@ -44,18 +44,19 @@ these migrations.
...
-| 2.16.x      | -    | -    | ✓    | ✓     | ✓      | ✓      |
-| 2.15.x      | -    | -    | -    | ✓     | ✓      | ✓      |
-
-\* Kubernetes 1.17 and 1.18 releases have reached End-of-Life (EOL). We
+| KKP version | 1.22 | 1.21 | 1.20 | 1.19\* | 1.18\*\* | 1.17\*\* |
+| ----------- | ---- | ---- | ---- | ------ | -------- | -------- |
+| 2.18.x      | ✓    | ✓    | ✓    | ✓      | -        | -        |
+| 2.17.x      | -    | ✓    | ✓    | ✓      | ✓        | -        |
+| 2.16.x      | -    | -    | ✓    | ✓      | ✓        | ✓        |
+| 2.15.x      | -    | -    | -    | ✓      | ✓        | ✓        |
```
Comment (SimonTheLeg) [inline content/kubermatic/v2.18/architecture/requirements/support_policy/supported_versions/_index.en.md:52]: I think this table is missing the entry 2.19

**Review by SimonTheLeg — CHANGES_REQUESTED** [review]

File: content/kubermatic/v2.18/architecture/requirements/support_policy/supported_versions/_index.en.md (hunk 2 above)
Comment (dermorz) [reply content/kubermatic/v2.18/architecture/requirements/support_policy/supported_versions/_index.en.md:52]: These are the 2.18 docs. (See file path)

**Review by themue — APPROVED** [review]
/lgtm
/approve

**Review by SimonTheLeg — APPROVED** [review]

**Pull request merged**